packages/analytics-browser/src/storage/local-storage.ts

```diff
@@ -1,8 +1,26 @@
 import { getGlobalScope } from '@amplitude/analytics-client-common';
 import { BrowserStorage } from './browser-storage';
+import { Logger } from '@amplitude/analytics-types';
+import { MAX_ARRAY_LENGTH } from '../constants';
```

Comment:
Since it's not configurable, I would probably declare `MAX_ARRAY_LENGTH`in `local-storage.ts`.

Reply:
I was thinking about it for a sec. Move it to `local-storage.ts` without export.